install/installer/pkg/config/v1/experimental/experimental.go

```diff
@@ -232,6 +232,7 @@ type ServerConfig struct {
 	EnableLocalApp                    *bool             `json:"enableLocalApp"`
 	RunDbDeleter                      *bool             `json:"runDbDeleter"`
 	DisableWorkspaceGarbageCollection bool              `json:"disableWorkspaceGarbageCollection"`
+	InactivityPeriodForRepos          *int              `json:"inactivityPeriodForRepos"`
```

Comment:
What's the integer unit? I'd recommend specifying the units in the name make it easy for the reader to grasp.

Reply:
That is also my preference, but implies refactoring all occurrences. I can do that in this PR.

Reply:
While also renaming the JSON variable would be nice, you can actually Rename just the Go variable and keep the JSON name the same.

Reply:
I prefer to keep consistent names across the entire code base, because otherwise I never know what should be called what and where (I've been there 🙈).

Thus, I've clarified every occurrence of this variable. Explicit variable names go a long way towards code understandability and peace of mind. 🧘